Source/Scene/PointCloud3DTileContent.js

```diff
@@ -898,6 +913,7 @@ define([
 
         var vs = 'attribute vec3 a_position; \n' +
                  'varying vec4 v_color; \n' +
+                 'uniform float u_pointAttenuationMaxSize; \n' +
```

Comment:
I think it might be a bit easier to keep `u_pointSizeAndTilesetTime` as is because the `PointCloudPostProcessor` directly edits the uniform map, replacing the `u_pointAttenuationMaxSize` uniform function with a function that just returns the attenuation bound. It can't edit `u_pointSizeAndTilesetTimeAndAttenuation` as cleanly because of the other two attributes (point size and tileset time) that are returned by the function.